frontend/packages/wizard/src/wizard.tsx

```diff
@@ -136,28 +136,26 @@ const Wizard = ({ heading, width = "default", dataLayout, children }: WizardProp
           </WizardContext.Provider>
         </DataLayoutContext.Provider>
         <Grid container justify="center">
-          {((state.activeStep === lastStepIndex && !isLoading) || hasError) && (
```

Comment:
noticed that we removed the `isMultistep` check in https://github.com/lyft/clutch/pull/1917 which was previously there(https://github.com/lyft/clutch/blame/917d40a9001d1aee8e28692d74a2e01f80383ccc/frontend/packages/wizard/src/wizard.tsx). Now that we're globally enabling the feedback, going to keep it to the original logic in case there were unknowns unknowns that didn't break before.

Reply:
cc @jdslaugh in case there's context I'm missing for the change

Reply:
Yeah, so the context for that is based on isMultistep. In this change we will only show feedback if we're a multistep wizard workflow and on the last step. So if there's a single step wizard (not sure if there is or will be), we'll never see it even if it's complete.

That was the reason for the adjustment in logic and moving the start over button under hasError/isMultistep

Reply:
ah got it! will undo my changes